aict_tools/configuration.py

```diff
@@ -108,30 +105,20 @@ def from_yaml(cls, configuration_path):
             return cls(yaml.load(f))
 
     def __init__(self, config):
-        self.has_multiple_telescopes = config.get('multiple_telescopes', False)
-        self.runs_key = config.get('runs_key', 'runs')
-        self.true_energy_column = config.get('true_energy_column')
-        self.size_column = config.get('size')
-
-        if self.has_multiple_telescopes:
-            self.telescope_events_key = config.get('telescope_events_key', 'events')
-            self.array_events_key = config.get('array_events_key', 'array_events')
-
-            self.array_event_id_column = config.get(
-                'array_event_id_column', 'array_event_id'
-            )
-            self.run_id_column = config.get('run_id_column', 'run_id')
-        else:
-            self.telescope_events_key = config.get('telescope_events_key', 'events')
-            self.run_id_column = config.get('run_id_column', 'run_id')
-
-            self.array_events_key = None
-            self.array_event_id_column = None
-
-        self.energy_unit = u.Unit(config.get('energy_unit', 'GeV'))
+        self.data_format = config.get('data_format', "simple")
+        self.datamodel_version = config.get('datamodel_version', "v1.1.0")
```

Comment:
Why is this a config option? This can we not just read it from the input files?

Reply:
We could, but we have all the columns defined in the AICTConfig object, which does not have a file associated. Not sure what the best solution would be here